sgtm/apollo/README.md

```diff
@@ -0,0 +1,196 @@
+*   A prepared spreadsheet that has values with 3 dimensions. 
+   (e.g. the related job at the bottom. age group at the left tap 
+    and the address at the top)
+    ![Spreadsheet](docs/img/spreadsheet_ss.png "Spreadsheet")
+
+
+### Prepare the environment
+
+Clone the repository to a local folder in your current environment (we recommend
+using cloud shell for the deployment as it provides all required utilities out
+of the box), using this command:
+
+```sh
+git clone https://github.com/google/gps-apollo
+```
+
+If you are not using cloud shell, you must install the
+[gcloud CLI](https://cloud.google.com/sdk/docs/install) in your environment.
+
+### Configure Tag Manager
+
+#### Import the template
+
+Download the template file [here](gtm/variable_template.tpl).
+
+From the Google Tag Manager UI, select your server container, go to the
+`Templates` section, and click on `New`:
+
+![New Template](docs/img/new_template.png "New Template")
+
+Then, click on the top right options button, and select `Import`:
+
+![Import Template](docs/img/import_template.png "Import Template")
+
+Select the file you downloaded and continue. Then, click on `Save` and close the
+template using the top left close button.
+
+#### Create the variable
+
+Now, from the Google Tag Manager UI, go to the `Variables` section, and click on
+`New`:
+
+![New Variable](docs/img/new_variable.png "New Variable")
+
+Click on `Variable Configuration`to select a variable type:
+
+![Variable Configuration](docs/img/variable_configuration.png "Variable Configuration")
+
+And select `gps-apollo`:
+
+![Select gps apollo](docs/img/select_gps_apollo.png "Select gps apollo")
+
+Then, fill in the details of your environment in the configuration screen:
+
+![Variable Details](docs/img/variable_details.png "Variable Details")
+
+
+Finally, to add the predicted value coming from Spreadsheet to the signup or
+conversion event, open the appropriate Tag and replace the conversion value (or
+equivalent) field with the variable you just created.
+
+For example, in Google Ads Conversion Tracking, you can use the `Conversion
+Value` field:
+
+![Google Ads Conversion Tracking](docs/img/google_ads_conversion.png "Google Ads Conversion Tracking")
+
+## Permissions
+
+The template uses sGTM's `getGoogleAuth` API, which uses Google Cloud's
+[Application Default Credentials](
+https://cloud.google.com/docs/authentication/application-default-credentials)
+to access the service account running sGTM.
+
+Open up [IAM](https://console.cloud.google.com/iam-admin/iam) in Google Cloud,
+and copy your app engine or cloud run service account.
+
+![Permission GCP](docs/img/permission_gcp.png "Permission GCP")
+
+and paste it into the spreadsheet for editor.
+
+![Permission Spreadsheet](docs/img/permission_spreadsheet.png "Permission Spreadsheet")
+
+## Disclaimers
+
+**This is not an officially supported Google product.**
+
+*Copyright 2023 Google LLC. This solution, including any related sample code or
```

Comment:
Nit: 2024

Reply:
Yes, I updated 2024 now.
Thank you for catching it!!